doc/aws-batch.md

```diff
@@ -24,6 +24,13 @@ directory.
 [AWS Batch]: https://aws.amazon.com/batch/
 [`zika-tutorial/` directory]: https://github.com/nextstrain/zika-tutorial
 
+### Requesting resources
+
+By default AWS Batch jobs request 4 vCPUs and 7200MB of memory. Generally, when
+running Snamemake, the `--jobs` option should be matched to the requested number
+of vCPUs. These defaults can be overridden by specifying `--aws-batch-cpus` and
+`--aws-batch-memory`, for instance `--aws-batch-cpus=8` and
+`--aws-batch-memory=14800`.
```

Comment:
It would be good to mention that:

* AWS Batch may not be able to run your job if your configured compute environment doesn't allow for instances with the CPUs/memory you request.

* Defaults for the number of CPUs and amount of memory can be set using environment variables and/or the config file.

Reply:
There's also a k → m typo in Snakemake.